solana-library/minimum_balance.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: Apache-2.0
+
+// Disclaimer: This library provides functions for working with storage rent. Although it is production ready,
+// it has not been audited for security, so use it at your own risk.
+
+/// Default rental rate in lamports/byte-year.
+///
+/// This calculation is based on:
+/// - 10^9 lamports per SOL
+/// - $1 per SOL
+/// - $0.01 per megabyte day
+/// - $3.65 per megabyte year
+uint64 constant DEFAULT_LAMPORTS_PER_BYTE_YEAR = 1_000_000_000 / 100 * 365 / (1024 * 1024);
+
+/// Default amount of time (in years) the balance has to include rent for the
+/// account to be rent exempt.
+/// Note that rent is currently not
```

Comment:
`is currently not` what?

Reply:
I don't know how that crept in. Thanks for spotting.